style.css

```diff
@@ -0,0 +1,160 @@
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.header-background-mobile {
+  width: 375px;
+  height: 718px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  background-color: #ffff;
+}
+
+.mylogo a {
+  position: absolute;
+  width: 150px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  top: 50px;
+  left: 0;
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 1000;
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 20px;
+  flex: none;
+  text-align: center;
+}
+
+.union {
+  position: absolute;
+  top: 45px;
+  left: 320px;
+}
+
+.primarytext {
+  width: 300px;
+  height: 150px;
+  top: 200px;
+  left: 24px;
+  color: #172b4d;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  position: absolute;
+  flex: none;
+  flex-grow: 0;
+}
+
+.bodytext {
+  width: 327px;
+  height: 146px;
+  font-style: normal;
+  font-weight: 400;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  position: absolute;
+  top: 370px;
+  left: 24px;
+  color: #344563;
+  flex: none;
+}
+
+#supportingtext {
+  width: 150px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  size: 16px;
+  line-height: 24px;
+  position: absolute;
+  top: 530px;
+  left: 24px;
+}
+
+.socialmediaicons {
+  position: relative;
+  top: 570px;
+  left: -18px;
+  list-style-type: none;
+  align-items: center;
+  justify-content: space-between;
+  display: flex;
+  width: 150px;
+}
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT** I think it would be better if you remove all the fixed widths(e.g:` max-width or width: 375px, 327px,...`) in your code base to make your page be responsive in every screen size. If you use those fixed sizes it will be difficult for you when you will start the **Desktop version** in the next projects.